dev/cognitiveservices/data-plane/Language/analyzeconversations.json

```diff
@@ -2096,6 +2105,201 @@
+          "$ref": "#/definitions/TranscriptContentType"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "common.json#/definitions/PreBuiltTaskParameters"
+        }
+      ]
+    },
+    "AnalyzeConversationSentimentResult": {
+      "type": "object",
+      "description": "Result from the sentiment analysis operation performed on a list of conversations.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/AnalyzeConversationJobResult"
+        }
+      ],
+      "properties": {
+        "results": {
+          "$ref": "#/definitions/ConversationSentimentResults"
+        }
+      },
+      "required": [
+        "results"
+      ],
+      "x-ms-discriminator-value": "ConversationalSentimentResults"
+    },
+    "ConversationSentimentResults": {
+      "type": "object",
+      "description": "The result from sentiment analysis operation for each conversation.",
+      "required": [
+        "conversations"
+      ],
+      "properties": {
+        "conversations": {
+          "type": "array",
+          "items": {
+            "allOf": [
+              {
+                "$ref": "#/definitions/ConversationSentimentResult"
+              },
+              {
+                "$ref": "#/definitions/ConversationResultBase"
+              }
+            ]
+          }
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "common.json#/definitions/PreBuiltResult"
+        }
+      ]
+    },
+    "ConversationSentimentResult": {
+      "description": "The result from sentiment analysis operation for each conversation item.",
+      "required": [
+        "conversationItems"
+      ],
+      "properties": {
+        "conversationItems": {
+          "description": "Enumeration of PII detection and redaction operation results for all the conversation items in a conversation.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ConversationSentimentItemResult"
+          }
+        }
+      }
+    },        
+    "ConversationSentimentItemResult": {
+      "type": "object",
+      "required": [
+        "id",
+        "participantId",
+        "sentiment",
+        "confidenceScores",
+        "sentences"
+      ],
+      "properties": {
+        "id": {
+          "type": "string"
+        },
+        "participantId": {
+          "type": "string"
+        },
+        "sentiment": {
+          "$ref": "#/definitions/Sentiment"
+        },                
+        "confidenceScores": {
+          "type": "object",
+          "$ref": "#/definitions/SentimentConfidenceScores"
+        },
+        "sentences": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SentenceSentiment"
+          }
+        }
+      }
+    },
+    "Sentiment": {
+      "type": "string",
+      "description": "Predicted sentiment.",
+      "enum": [
```

Comment:
Should the sentiment enum be factored out into a common definition between conversation and [text](https://github.com/Azure/azure-rest-api-specs/blob/a9e895ccfe29d0646795f7ff1cb78e185bd09529/specification/cognitiveservices/data-plane/Language/stable/2022-05-01/textanalytics.json#L1117)?

Reply:
yeah I agree